pkg/signoz/handler.go

```diff
@@ -2,6 +2,7 @@ package signoz
 
 import (
 	"github.com/SigNoz/signoz/pkg/factory"
+	flaggerPkg "github.com/SigNoz/signoz/pkg/flagger"
```

Comment:
```suggestion
 "github.com/SigNoz/signoz/pkg/flagger"
```

Why not just `Flagger`?